### ISSUE TITLE
Please add an airgapped install option

### ISSUE THREAD

**Issue by devoptimist** [habitat-sh/on-prem-builder#117]
For customers who operate in an airgapped environment it would be nice to offer an installer option similar to chef automates airgapped bundle option https://automate.chef.io/docs/airgapped-installation/.
This uses the chef-automate cli tool to create a bundle of packages needed to bootstrap an a2 installation.

**Comment by lhasadreams** [comment]
Can someone give me a view as to whether this is being considered please.

**Comment by chefsalim** [comment]
Yes this is something we'd like to add and will consider further.

**Comment by devoptimist** [comment]
We made this temporary work arround for generating airgapped bundles. https://github.com/devoptimist/habitat-bundler?files=1 (wip)
